libs/vkd3d/vkd3d_private.h

```diff
@@ -170,6 +171,7 @@ struct vkd3d_vulkan_info
     bool NV_shader_subgroup_partitioned;
     bool NV_memory_decompression;
     bool NV_device_generated_commands_compute;
+    bool NV_low_latency2;
```

Comment:
Needs to validate somehow that the extension version is at least 2 due to the breaking API change.

Reply:
I added an additional commit to this PR that checks the extension version in has_extension.